src/routes/game/components/ChooseWhetherToCounterDialog.vue

```diff
@@ -42,21 +42,23 @@
 
     <template #actions>
       <v-btn
-        data-cy="decline-counter-resolve"
-        color="surface-1"
-        variant="outlined"
-        class="mr-4"
-        @click="resolve"
+        data-cy="counter"
+        color="newPrimary"
+        variant="flat"
```

Comment:
Had some back and forth about the button color and style. Ultimately opting for `newPrimary` pink in order to:
1. Visually differentiate the buttons as much as possible
2. Avoid favoring one option over the other as both are important